### PR TITLE
improved wiki

### PR DESCRIPTION
**Pull request by gurungrahul2**
added botfather link

-----
[View rendered README.md](https://github.com/gurungrahul2/git-profiler-bot/blob/readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ $ npm install serverless -g
 $ npm install
 
 # Get a bot from Telegram, sending this message to @BotFather
+Go to https://telegram.me/botfather
```
[the remark this answers is not in the document]
Comment (gurungrahul2) [reply README.md]: sure.

[the remark this answers is not in the document]
Comment (gurungrahul2) [reply README.md]: @aashutoshrathi actually we can not make things clickable inside ```sh < content >```sh

[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply README.md]: Okay then at least make it short.

[the remark this answers is not in the document]
Comment (gurungrahul2) [reply README.md]: done.

File: README.md [hunk 2]
```diff
@@ -26,10 +26,11 @@ $ npm install serverless -g
 $ npm install
 
 # Get a bot from Telegram, sending this message to @BotFather
+Go to https://t.me/BotFather
 $ /newbot
 
 # Put the token received into a file called serverless.env.yml, like this
-$ cat serverless.env.yml
+$ cat >serverless.env.yml
```
Comment (aashutoshrathi) [inline README.md]: Why added `>`

Comment (gurungrahul2) [reply README.md]: actually in ubuntu cat filename doesn't works for creating new file, it is used for viewing existing new file, cat >filename creates new file and takes input along with it to be put into file.

Comment (gurungrahul2) [reply README.md]: https://www.geeksforgeeks.org/cat-command-linux-examples/

**Pull request merged**